src/ai/chat.ts

```diff
@@ -0,0 +1,44 @@
+import SdkBase from '../sdkBase';
+import Ainft721Object from '../ainft721Object';
+import Assistants from './assistants/assistants';
+import { ChatConfigureParams, TransactionResult } from '../types';
+import {
+  buildSetTransactionBody,
+  validateObject,
+  validateObjectOwnership,
+  validateService,
+  validateServiceName,
+} from '../util';
+
+export default class Chat extends SdkBase {
```

Comment:
Instead of inheriting from SdkBase, how about adding an explicit constructor e.g. 
```
  constructor(ain: Ain, ainize: Ainize) {
    this.ain = ain;
    this.ainize = ainize;
};
```
for readability?

Reply:
Done!